data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+
+mission "Heliarch Expedition 4"
+	name "Extraterritorial Venture"
+	description "Escort a Heliarch ship through human space and into the wormhole in Deneb, so that they can further investigate the broken ringworld."
+	landing
+	source "Ablub's Invention"
+	destination "Ruin"
+	to offer
+		has "Heliarch Expedition 3: done"
+	on offer
+		conversation
+			`The Punisher's captain has some equipment loaded onto the ship, and the mechanics do one last maintenance check on the ship before it leaves home territory. The captain tells you they will follow your lead when you launch. Every other minute, he looks nervously at the sky.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Heliarch"
+		ship "Heliarch Punisher (Fuel)" "Eldest Inspector"
+	on visit
+		dialog `You've reached <planet> but left the Heliarch ship behind. Best depart and wait for them to arrive.`
+	on complete
+		conversation
+			`The Punisher's captain had you follow his ship all over the planet after they refueled in the planet's odd facilities. Apparently they want an isolated place to set up their camp here, somewhere they wouldn't be easily spotted.`
+			`	They settle for a cavern on the way up a mountain, overlooking a foggy valley. A river is faintly visible through the dense fog. As they unload the cargo and start setting up, the captain comes to you and says that they'll meet you in the spaceport in a few hours to discuss "the acquisition of supplies."`
+
+mission "Heliarch Expedition 5"
+	name "Settlement Supplies"
+	description "Buy an nGVF-CC Fuel Cell, an LP144a Battery Pack, and a Water Coolant System, then deliver them to the Heliarch agents on Ruin for them to set up their camp."
+	source "Ruin"
+	to offer
+		has "Heliarch Expedition 4: done"
+	on offer
+		conversation
+			`You've been waiting around for hours in the "spaceport" here when the Punisher finally comes back. The captain, a young Arach, comes to meet you, taking worried glances at the sky on his way. "Ready to do some shopping, are you, Captain?"`
+			choice
+				`	"Wasn't I gonna get some supplies for you? Figured you meant I just had to pick them up from Coalition space."`
+				`	"Why do you even need supplies? Wasn't your cargo full with those crates?"`
+			`	"A most... exquisite opportunity, this is. Very... excited, our researchers are. To keep them and the operation secure, I was tasked," he says. "Filled to the brim with only laboratory utensils and advanced equiment, our cargo was, unfortunately."`
+			`	He goes over how they lack basic equipment to set up a proper camp and further details on how all the Punisher was carrying was the technology that will be used to study the ringworld, so they have no means of sustaining the camp here in the long run.`
+			choice
+				`	"You came here without supply or equipment? What the hell were you thinking?"`
+					goto nosupply
+				`	"Alright, what is it that you need?"`
+					goto needs
```

Comment:
Ah yes, the coalition can make sure to build a weather control system for a 1 in many million odds chance of a minor inconvenience but they can't supply their ship for an expedition properly, or communicate the need for a resupply chain to be established in advance?

Reply:
A solution for this is that the researchers find the equipment missing (resistance stole the equipment), and ask you to fetch some replacements for them.